app/src/main/java/org/mifos/mobile/ui/beneficiary/presentation/BeneficiaryAddOptionsFragment.kt

```diff
@@ -37,25 +45,27 @@ class BeneficiaryAddOptionsFragment : BaseFragment() {
         container: ViewGroup?,
         savedInstanceState: Bundle?,
     ): View {
-        _binding = FragmentBeneficiaryAddOptionsBinding.inflate(inflater, container, false)
-        setToolbarTitle(getString(R.string.add_beneficiary))
-        return binding.root
+        return ComposeView(requireContext()).apply {
+            setViewCompositionStrategy(ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed)
+            composeView = this
+        }
     }
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        composeView.setContent {
 
-        with(binding) {
-            llAddBeneficiaryManually.setOnClickListener {
-                addManually()
-            }
-            llAddBeneficiaryQrcode.setOnClickListener {
-                addUsingQrCode()
-            }
-            binding.llUploadBeneficiaryQrcode.setOnClickListener {
-                addByImportingQrCode()
+            MifosMobileTheme {
```

Comment:
You can directly use in above onCreateView function and remove composeView field

Reply:
![Screenshot (119)](https://github.com/openMF/mifos-mobile/assets/90254566/cc77e4ac-2f15-4dad-aae7-ff42f9cafa9b)